easybuild/easyconfigs/u/UCX/UCX-1.8.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.8.0'
+
+homepage = 'http://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openucx/ucx/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+patches = [
+    'UCX-1.7.0_binutils_2.34_api_fix.patch',
+]
+checksums = [
+    'e400f7aa5354971c8f5ac6b881dc2846143851df868088c37d432c076445628d',  # ucx-1.8.0.tar.gz
+    'c09ebe4d734d520ae23f56d95ba0b91e464a42ccbaf435675424515ebd3fa3a9',  # UCX-1.7.0_binutils_2.34_api_fix.patch
+]
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+]
+
+osdependencies = [('libibverbs-dev', 'libibverbs-devel', 'rdma-core-devel')]
```

Comment:
The compilation went fine but I had runtime issues due to missing system libraries. See https://github.com/openucx/ucx/issues/4883
Therefore, I would add a comment to avoid the hassle for users with Mellanox devices
```suggestion
# Mellanox devices also need mlnx-ofed-* and libmlx5-devel for mlx5 driver (see issue openucx/ucx#4883)
osdependencies = [('libibverbs-dev', 'libibverbs-devel', 'rdma-core-devel')]
```